src/utils/loadScript.ts

```diff
@@ -0,0 +1,12 @@
+type ScriptProps = "src" | "id";
+type LoadScriptProps = Record<ScriptProps, string>;
+
+export function loadScript(options: LoadScriptProps) {
```

Comment:
Is there a reason for the ES5 syntax? On the next line you are using `const` I suppose this can become an arrow function and keep it consistent.

Reply:
Same question/logic applies for the function `initializeZendesk`

Reply:
Good addition, changed that